test/t8_cmesh_generator/t8_cmesh_example_sets.hxx

```diff
@@ -52,8 +52,12 @@ auto pretty_print_base_example = [] (const testing::TestParamInfo<cmesh_example_
 auto pretty_print_base_example_scheme = [] (const testing::TestParamInfo<std::tuple<int, cmesh_example_base *>> &info) {
   std::string name;
   std::get<1> (info.param)->param_to_string (name);
-  name += std::string ("_scheme_") + std::to_string (std::get<0> (info.param));
-  name += std::string ("_") + std::to_string (info.index);
+  if (std::get<0> (info.param) == 0) {
+    name += std::string ("_Default_Scheme");
+  }
+  else {
+    name += std::string ("_Standalone_Scheme");
+  }
```

Comment:
Can you maybe use this char here? Then we do not define it twice
https://github.com/DLR-AMR/t8code/blob/77af37fb12e1fab7d20997d452e337cb22d5180f/test/t8_gtest_schemes.hxx#L44